lib/debug/breakpoint.rb

```diff
@@ -389,11 +389,20 @@ def enable
 
     def override klass
       sig_method_name = @sig_method_name
-      klass.prepend Module.new{
-        define_method(sig_method_name) do |*args, **kw, &block|
-          super(*args, **kw, &block)
-        end
-      }
+
+      if RUBY_VERSION.to_f <= 2.6
```

Comment:
please check the condition outside of definition?

```
if RUBY_VERSION.to_f <= 2.6
  def override klass ...
else
  ...
end
```

Reply:
👍 updated